app/src/main/java/com/innowisegroup/datetimepicker/DateTimePickerDialog.kt

```diff
@@ -9,50 +9,62 @@ import android.widget.Button
 import android.widget.LinearLayout
 import androidx.core.content.res.ResourcesCompat
 import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
 import androidx.fragment.app.FragmentManager
+import androidx.viewpager2.widget.ViewPager2
 import com.google.android.material.tabs.TabLayout
+import com.google.android.material.tabs.TabLayoutMediator
 
-class DateTimePickerDialog(
-        initialLocalDateTime: LocalDateTime?,
-        private val minLocalDateTime: LocalDateTime?,
-        private val maxLocalDateTime: LocalDateTime?,
-        private val wrapSelectionWheel: Boolean,
-        private val withOnlyDatePicker: Boolean,
-        private val withOnlyTimePicker: Boolean
-) : DialogFragment() {
+class DateTimePickerDialog : DialogFragment() {
 
     private var tabLayout: TabLayout? = null
 
-    private var viewPager: CustomViewPager? = null
+    private var viewPager: ViewPager2? = null
 
     private val okClickCallback: OkClickCallback? = null
 
     private val cancelClickCallback: CancelClickCallback? = null
 
-    private var adapter: PagerAdapter? = null
-
     private var initialLocalDateTime: LocalDateTime? = null
+    private var initialLocalTime: LocalTime? = null
 
     private var timePickerFragment: FragmentTimePicker? = null
     private var datePickerFragment: FragmentDatePicker? = null
 
-    init {
-        this.initialLocalDateTime = initialLocalDateTime
-                ?: LocalDateTime().now()
-    }
+    private var minLocalDateTime: LocalDateTime? = null
+    private var maxLocalDateTime: LocalDateTime? = null
+    private var wrapSelectionWheel: Boolean = false
+    private var fragmentToCreate: FragmentToCreate = FragmentToCreate.BOTH
+
+    private var currentTimeStub = "99:99"
+    private var currentDateStub = "99-99-99"
 
     override fun onCreateView(
-            inflater: LayoutInflater,
-            container: ViewGroup?,
-            savedInstanceState: Bundle?
```

Comment:
Seems like you and Andrew have different formatting? Make a sync and check if you have the same code formatting to prevent unnecessary code changes